packages/discord.js/src/structures/GuildMember.js

```diff
@@ -53,6 +53,11 @@ class GuildMember extends Base {
      */
     this.communicationDisabledUntilTimestamp = null;
 
+    /**
+     * The role ids of this member
```

Comment:
```suggestion
     * The role ids of the member
```
Could probably be worded more differently